controller/booksController.js

```diff
@@ -1 +1,33 @@
 const books = require('../modules/books');
+
+function bookRegistration(req) {
+  const { bookName } = req.body;
```

Comment:
You can desestructure the req.body only once:
const { bookName, authorName, bookRelease ... } = req.body;